README.md

```diff
@@ -56,22 +56,94 @@ Code quality:
     ```
 
 1. Start app with required services
-    ```shell script
+    ```shell
     make runserver
     ```
 
 1. Service will be available at `http://127.0.0.1:8042/`
 
+## Start the project for developing without Docker
+1. Clone the repo and go to the project directory
+    ```shell
+    git clone git@github.com:Uma-Tech/parrot.git && \
+    cd parrot
+    ```
+
+1. Install dependencies
+    ```shell
+    poetry install
+    ```
+
+1. Start databases  
+    _You can start databases in any other way_
+   ```shell
+    # start postgres
+   docker run \
+       --name postgres \
+       -d \
+       --rm \
+       -e POSTGRES_PASSWORD=parrot \
+       -e POSTGRES_USER=parrot \
+       --network host \
+       -v parrot_db:/var/lib/postgresql/data \
+       postgres
+   
+    # start redis
+   docker run \
+       --name redis \
+       -d \
+       --rm \
+       --network host \
+       redis 
+    ```
+
+1. Setting require environment variables  
+   _Alternative, you can create a local `.env` file with the variables_
+    ```shell
+    export PARROT_DB_HOST=127.0.0.1
+    export PARROT_SECRET_KEY=NO_SECRET
+    ```
+
+1. Build static
+    ```shell
+    poetry run python manage.py collectstatic
+    ```
+
+1. Apply database migrations
+    ```shell
+    poetry run python manage.py migrate
+    ```
+
+1. Create a django superuser
+    ```shell
+    poetry run python manage.py createsuperuser
+    ```
+
+1. Start the django app
+    ```shell
+    poetry run python manage.py runserver
+    ```
+
+1. Start the celery worker  
+   _Run in a separate terminal window_
+    ```shell
+    poetry run celery -A parrot worker -l INFO
+    ```
+    
+
 ## System requires
```

Comment:
I know it's not from this PR, but just noticed
```suggestion
## System requirements
```

Reply:
Yes, but thanks anyway!